client/app/components/items-list/components/EmptyState.jsx

```diff
@@ -2,10 +2,10 @@ import React from 'react';
 import { BigMessage } from '@/components/BigMessage';
 
 // Default "list empty" message for list pages
-export default function LoadingState() {
+export default function LoadingState(props) {
```

Comment:
Prolly rename to `EmptyListState`?

Reply:
It's copy-paste :man_facepalming: Thanks!